tutorials/polkadot-sdk/build-a-parachain/prepare-relay-chain.md

```diff
@@ -0,0 +1,202 @@
+
+## Build a Local Relay Chain
+
+To build a local relay chain, follow these steps:
+
+1. Clone the most recent release branch of the Polkadot SDK repository to prepare a stable working environment
+
+    ```bash
+    git clone --depth 1 --branch polkadot-stable2407-2 https://github.com/paritytech/polkadot-sdk.git
+    ```
+
+    !!! note
+        The branch `polkadot-stable2407-2` is used in this tutorial since it is the branch that contains the latest stable release of the Polkadot SDK. You can find the latest release of the Polkadot SDK on the [Release](https://github.com/paritytech/polkadot/releases){target=\_blank} tab on the Polkadot GitHub repository.
+
+    !!! note
+        Note that the `--depth 1` flag is used to clone only the latest commit of the branch, which speeds up the cloning process.
+
+2. Change the directory to the Polkadot SDK repository
+
+    ```bash
+    cd polkadot-sdk
+    ```
+
+3. Build the relay chain node by running the following command:
+
+    ```bash
+    cargo build --release
+    ```
+
+    !!! note
+        Depending on your machine’s specifications, the build process may take some time.
+
+4. Verify that the node is built correctly by running the following command:
+
+    ```
+    ./target/release/polkadot --version
+    ```
+
+If command-line help is displayed, the node is ready to configure.
+
+## Relay Chain Configuration
+
+Every Substrate-based chain requires a [chain specification](https://docs.substrate.io/build/chain-spec/){target=\_blank}. The relay chain network’s chain specification provides the same configuration settings as the chain specification does for other networks. Many of the chain specification file settings are critical for network operations. For example, the chain specification identifies peers participating in the network, keys for validators, boot node addresses, and other information.
+
+### Sample Chain Configuration
+
+The local relay chain uses a sample chain specification file with two validator relay chain nodes—Alice and Bob—as authorities for this tutorial. Because a relay chain must have at least one more validator node running than the total number of connected parachain collators, you can only use the chain specification from this tutorial for a local relay chain network with a single parachain.
+
+If you wanted to connect two parachains with a single collator each, you would need to run three or more relay chain validator nodes. You would need to modify the chain specification and hard-code additional validators to set up a local test network for two or more parachains.
+
+### Plain and Raw Chain Specification
+
+The chain specification file is available in two formats: a JSON file in plain text and a JSON file in SCALE-encoded raw format.
+
+- [Plain sample relay chain spec](https://docs.substrate.io/assets/tutorials/relay-chain-specs/plain-local-chainspec.json/){target=\_blank}
```

Comment:
We should check if these chain specs are up to date. If not, we can reference others